Source/Core/VideoCommon/TextureDecoder_Common.cpp

```diff
@@ -434,6 +434,11 @@ static inline u32 MakeRGBA(int r, int g, int b, int a)
   return (a << 24) | (b << 16) | (g << 8) | r;
 }
 
+static constexpr int DXTBlend(int v1, int v2)
```

Comment:
I wrote a Wii homebrew that displayed the hex values from decoding on the screen (see the gist link in the PR description). Then I ran it, took a photo of my TV, and then typed a number of test values in by hand and checked them twice.

You can have a look at how Dolphin's real hardware tests work: https://github.com/dolphin-emu/hwtests - I had problems with the network stuff, so I used the libogc example "template.c", which prints to the screen. I'll hopefully work out the network issues to make it easier next time.